starport/services/chain/serve.go

```diff
@@ -552,8 +557,13 @@ func (c *Chain) runFaucetServer(ctx context.Context, faucet cosmosfaucet.Faucet)
 		return err
 	}
 
+	host := conf.Faucet.Host
```

Comment:
Can we move this part related to determining address for the faucet to `starporconf.FaucetHost()`, and reuse it here?